kustomize/install/singlenamespace/manager-selector.yaml.yaml

```diff
@@ -0,0 +1,5 @@
+- op: remove
```

Comment:
🤔 This file and patch directive are in both `default` and `singlenamespace`. Should it move into `manager`?

Or perhaps we drop this copy in `singlenamespace` and refer to it by path `../default/manager-selector.yaml`?

Reply:
Since `singelanamespace` and `default` are at the same level, they can't call files from each other -- we get the `file ___ is not in or below` the current folder.

Moving it to `manager` makes some sense to me, though sort of funny to have an operation in `manager` remove labels that aren't even being added in `manager`. For the purposes of this story, I think it makes sense to leave as is.